content/episodes/Best practices to build Docker images.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Best Practices for Building Docker Images"
+date: 2020-10-12T13:18:01+02:00
+lastmod: 2020-10-12T13:18:01+02:00
+episode: 17
```

Comment:
I'm a bit lost in the numbering.....

Reply:
I think we can drop latmod altogether. Not sure what is the purpose